### DISCUSSION TITLE
Tina Cloud not working, schema errors

### DISCUSSION THREAD

**Discussion by Coru89** [tinacms/tinacms#4768] [Help] [answered]
The CMS is working in my local. I just migrated an old Forestry WIP project to tina. When trying to take tina to prod via tina cloud, the CMS loads in prod, but no data is found and errors are thrown. 

![image](https://github.com/user-attachments/assets/915aa2c7-32a7-4747-bbee-ae609322db3c)

![image](https://github.com/user-attachments/assets/6697a8ac-28e8-4bd7-8e25-3d73cca6971d)

![image](https://github.com/user-attachments/assets/5f3244db-38b7-4bb5-9275-dcf3c3ea518e)

https://coreykeller.netlify.app/

Any ideas?

**Comment by Coru89** [comment]
linking to the repo: https://github.com/Coru89/coreykeller.ca

**Comment by ncn-ssw** [comment]
Hi @Coru89,

I have looked at the TinaCloud logs, and there several HTTP 405 errors (i.e. attempting to GET rather than POST) associated with your account from around 7 September. I'm not seeing other errors raised by TinaCloud.

How are you running your project locally?

What is the output you see when you run `npm start` locally?

**Comment by kldavis4** [comment, marked as answer]
@Coru89 I am able to replicate this (`Unexpected error checking schema: TypeError: Failed to fetch`) with your repository in Chrome, but it looks like it might be a browser issue. After deleting cookies, though, the issue goes away. Try deleting cookies or use incognito mode and let us know if that resolves it.

**Comment by Coru89** [comment]
Wow! Thanks to both of you guys for your response. I never thought to clear my cache for this issue. I didn't think the solution could be so simple but that did it. I was analyzing the schema looking for errors... but no, the cache refresh did it.

Cheers!
